contributions/catalog/catalog_rest_service.py

```diff
@@ -130,8 +130,7 @@ def login():
 
 
 # Step 2: User authorization, this happens on the provider.
-# "http://localhost:5050/contributions/catalog/auth/callback"
-@app.route("/contributions/catalog/auth/callback", methods=["GET"])
+@app.route("/catalog/auth/callback", methods=["GET"])
```

Comment:
The "+Contribute" button got redirected to the correct URL. The "Login" button had an issue. As discussed, will test the same after merging to dev instance.